src/Knp/Menu/Renderer/Renderer.php

```diff
@@ -62,8 +65,12 @@ private function htmlAttributesCallback(string $name, $value): string
     /**
      * Escapes an HTML value
      */
-    protected function escape(string $value): string
+    protected function escape(?string $value): string
```

Comment:
I'd rather add a `assert($item->getUri() !== null)` in `ListRenderer::renderLinkElement` than accepting null here